srcpkgs/python3-curtsies/template

```diff
@@ -1,16 +1,18 @@
 # Template file for 'python3-curtsies'
 pkgname=python3-curtsies
-version=0.3.0
-revision=6
-build_style=python3-module
-hostmakedepends="python3-setuptools"
-depends="python3-blessings python3-wcwidth"
+version=0.4.1
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3-blessed python3-cwcwidth"
+checkdepends="${depends} python3-pyte python3-pytest"
 short_desc="Curses-like terminal wrapper, with colored strings (Python3)"
 maintainer="Maurizio Porrato <maurizio.porrato@gmail.com>"
 license="MIT"
-homepage="https://github.com/thomasballinger/curtsies"
+homepage="https://pypi.org/project/curtsies/"
```

Comment:
```suggestion
homepage="https://github.com/bpython/curtsies"
```

Reply:
Yeah I should probably stop doing that. My reasoning was [this](https://github.com/void-linux/void-packages/pull/41386#discussion_r1059905844) btw.

Reply:
The rule of thumb is that the homepage should be whatever upstream considers to be the official homepage. If PyPI provides a link to the project homepage, use the target of that link. If the project page is a GitHub repo and the GitHub repo lists another link in the project description, use the target of that link instead (even if PyPI takes you to the GH page).